build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -0,0 +1,172 @@
+                      oneOf:
+                        - format: ipv4
+                        - format: ipv6
+                packet:
+                  type: object
+                  properties:
+                    ipv6Header:
+                      type: object
+                      properties:
+                        nextHeader:
+                          type: integer
+                          minimum: 0
+                          maximum: 65535
+                    transportHeader:
+                      type: object
+                      properties:
+                        udp:
+                          type: object
+                          properties:
+                            srcPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            dstPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                        tcp:
+                          type: object
+                          properties:
+                            srcPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            dstPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            flags:
+                              type: integer
+                              minimum: 0
+                              maximum: 255
+                timeout:
+                  type: integer
+                  minimum: 1
+                  maximum: 300
+                captureConfig:
+                  type: object
+                  anyOf:
+                    - properties:
+                        firstN:
+                          required: [number]
+                  properties:
+                    firstN:
+                      type: object
+                      properties:
+                        number:
+                          type: integer
+                          format: int32
+                fileServer:
+                  type: object
+                  properties:
+                    url:
+                      type: string
+                      pattern: 's{0,1}ftps{0,1}:\/\/[\w-_./]+:\d+'
+            status:
+              type: object
+              properties:
+                reason:
+                  type: string
+                phase:
+                  type: string
+                startTime:
+                  type: string
+                numCapturedPackets:
+                  type: integer
+                packetsFileName:
+                  type: string
+
+      subresources:
+        status: {}
+  scope: Cluster
+  names:
+    plural: packetcaptures
+    singular: packetcapture
+    kind: PacketCapture
+    shortNames:
+      - pcp
```

Comment:
do you think `pcap` may be a better name?

Reply:
sounds better!